src/content/docs/tutorial-innovation-and-growth/development-quality-mgmt.mdx

```diff
@@ -0,0 +1,166 @@
+* Have no inter- or external- dependencies.
+* Have simple and easy to maintain code.
+
+Unit test events should include the following attributes:
+* Test name
+* Commit being tested
+* Test duration
+* Test result (success/failure)
+  </Collapser>
+
+  <Collapser
+    id="kpi-code-coverage"
+    title="Code coverage KPI"
+  >
+Code coverage measures the amount of an application's code base that's subject to at least one unit test.
+
+**Goal:** Increase code coverage to 100%.
+
+**Best practices:** Follow the [unit test](#kpi-unit-test-success) best practices to ensure that code test coverage reaches the 100% goal.
+  </Collapser>
+
+  <Collapser
+    id="kpi-defect-volume"
+    title="Defect volume KPI"
+  >
+Defect volume measures the number of defects introduced into an application by a specific module of code.
+
+**Goal:** Reduce the number of new defects.
+**Best practices:** Identify the sources of new defects by relating them to specific services and code modules.
+
+Code defect events should include the following attributes:
+* Timestamp
+* Application
+* Code module
+* Reference to defect
+  </Collapser>
+</CollapserGroup>
+
+### Velocity [#kpi-velocity]
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-commit-volume"
+    title="Code commit volume KPI"
+  >
+Code commit rate measures the number of new code commits over time.
+
+**Goal:** Code commit rate should be correlated against defect volume to identify the optimal balance between velocity and stability.
+
+**Best practices:** Code commit events should include the following attributes:
+* Timestamp
+* User
+* Project
+* Repository
+* Branch
+* Commit ID or hash
+  </Collapser>
+</CollapserGroup>
+
+<Steps>
+    <Step>
+### Gather the required KPIs [#gather-kpis]
+
+The best place to start is by gathering your key performance indicators. To do this, you need to know the specific technology platforms that support your development processes, such as source code repositories and build/test automation platforms. Then, you'll need to identify methods for extracting each KPI's attributes and importing them into New Relic.
+
+You can find the KPIs and minimum-required attributes needed for this use case in the [key performance indicators](#key-perf-indicators) section above. Typically, you'll use your development toolchain's APIs to extract the KPIs and their attributes, then submit them to New Relic using the [custom events API](/docs/data-apis/ingest-apis/introduction-event-api/).
+
+Before starting any custom integration work, you should find if any applicable out-of-box integrations exist.    
+    </Step>
+    <Step>
+### Implement dashboards [#implement-dashboards]
+
+Our <InlinePopover type="dashboards" /> are the primary method this improvement process uses. They show your current KPIs and help you to identify the areas which require improvement. You can find sample dashboards [in the New Relic OMA resource center on GitHub](https://github.com/newrelic/oma-resource-center).
```

Comment:
i believe `areas which require improvement` should be `areas that require improvement` since `require improvement` is an ~~~essential clause that defines the particular area youre looking at

Reply:
Super good catch, thank you very much!